test/unit/dom/dom.js

```diff
@@ -200,9 +588,33 @@ describe('skatejs-named-slots dom', function () {
 
         host.innerHTML = processingInstruction;
         expect([processingInstruction, processingInstructionsAfterInnerHtml].indexOf(host.innerHTML)).to.be.above(-1);
+        expect(host.childNodes.length).to.equal(1);
+        expect(slot.getAssignedNodes().length).to.equal(1);
```

Comment:
Yeah, you're right. Happy with removing support for everything except for `Text` and `Element`.